lib/ngx/ssl.lua

```diff
@@ -261,6 +275,56 @@ function _M.set_priv_key(priv_key)
 end
 
 
+_M.PROTOCOL_SSLv2 = 0x0002
+_M.PROTOCOL_SSLv3 = 0x0004
+_M.PROTOCOL_TLSv1 = 0x0008
+_M.PROTOCOL_TLSv1_1 = 0x0010
+_M.PROTOCOL_TLSv1_2 = 0x0020
```

Comment:
I thikn we can just omit the `PROTOCOL_` prefix from these keys. They are not really needed IMHO.